playbook-website/config/initializers/content_security_policy.rb

```diff
@@ -9,6 +9,12 @@
   p.script_src  :self, :https, :unsafe_inline, :unsafe_eval
   p.worker_src  :self, "blob:"
 
+  if Rails.env.development?
+    p.frame_src   :self, "https://localhost:8000", "http://localhost:8000"
+  else
+    p.frame_src   :self, "https://pr3210.playbook.beta.gm.powerapp.cloud", "http://pr3210.playbook.beta.gm.powerapp.cloud"
```

Comment:
This will change. Just wanting to see if testing in a milano environment was even possible.